docs/getting_started/tutorial-msm.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>This is way to complicated and distracting for a tutorial. Why don't you just return <code></code>df.groupby("Period").Choice.value_counts(normalize=True)<code></code>? Maybe add <code></code>unstack()<code></code> to have a beautiful table.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>